alert.css

```diff
@@ -0,0 +1,113 @@
+#alert {
+    height: 87rem;
+    width: 100%;
+    padding: 2rem;
+    background-color:transparent;
+    font-family: "Hind", sans-serif;
+  }
+  .alert-heading{
+    font-size: 3rem;
+    color:black;
+  }
+  .component-para{
+    letter-spacing: 2px;
+    font-size: 1.5rem;
+    padding-right: 2rem;
+    line-height: 35px;
+    color:rgba(85, 85, 238, 0.678) ;

```

Comment:
Try to use CSS variables instead of using hardcoded color value. It will be good practice and easy to maintain.
Example: 
:root{
  --color-primary: rgba(85, 85, 238, 0.678);
}